.github/workflows/test.yaml

```diff
@@ -65,9 +65,13 @@ jobs:
           flags: be_integration_test_${{ matrix.tidb_version }}
           verbose: true
   e2e_test:
-    name: E2E
+    name: E2E Test
     runs-on: ubuntu-latest
     timeout-minutes: 10
+    strategy:
+      matrix:
+        # test latest features and compatibility of lower version
+        feature_flag: [6.0.0, 5.0.0]
```

Comment:
Maybe you will want this matrix, so that `tiup playground $TIDB_VERSION  --tiflash=0` can be covered and there is no magic check when running compatibility tests:

```yaml
strategy:
  matrix:
    include:
      - feature_version: 6.0.0
        tidb_version: nightly
        run_compat_test: no
      - feature_version: 5.0.0
        tidb_version: v5.0.0
        run_compat_test: yes
```